docs/api/en/textures/DepthTexture.html

```diff
@@ -119,6 +119,13 @@ <h3>[page:Texture.generateMipmaps .generateMipmaps]</h3>
 		<h3>[property:Boolean isDepthTexture]</h3>
 		<p>Read-only flag to check if a given object is of type [name].</p>
 
+		<h3>[property:String compareFunction]</h3>
```

Comment:
I don't think it's a string?

Reply:
b905e279516c89c4cb71d43f0d17a1f1077f470c

Reply:
It's an integer, correct?. Also, the default should be specified in the docs.

Reply:
There isn't any default value as otherwise it would involve that the compare function is enabled by default.

Reply:
The default is `null`. And the docs should explain the consequences of a `null` value.

Reply:
Should be fixed via 43c701c69aa7f5f690d521a9272f9d69f625c67f.